content/python/concepts/built-in-functions/terms/next/next.md

```diff
@@ -0,0 +1,71 @@
+---
+Title: 'next()'
+Description: 'Returns the next element from an iterable object.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Collections'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+The **`next()`** function returns the next element from an [iterator](https://www.codecademy.com/resources/docs/python/iterators) object.
+
+## Syntax
+
+```pseudo
+next(iterator_name, [default_parameter])
+```
+The required `iterator_name` parameter is the iterator object with the set of values. The `default_parameter` is optional and is printed if the end of the iterator is reached.
+
+> **Note:** If the next element is missing from the object, the `default_parameter` is returned. Without a set `default_parameter`, a `StopIteration` [error](https://www.codecademy.com/resources/docs/python/errors) is thrown.
+
+## Example
```

Comment:
Maybe it is a lack of concept on my side. What is the difference between Example and Codebyte Example? What I try to show with the first one is the most simple use you can do of the function over a list. The second one wants to extend the knowledge in case you miss the default parameter when the end of list is reached.

Reply:
I have just updated the file with your suggestions. Thanks.